remove-cache.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const cache = path.join(process.env.INIT_CWD, '.eslintcache');
```

Comment:
why not `process.cwd`?

Reply:
OK I looked it up and `process.cwd` is correct. I guess I got confused by others projects I worked on and thought that `INIT_CWD` was what we wanted. Good catch.